flink-container/kubernetes/README.md

```diff
@@ -29,7 +29,7 @@ In non HA mode, you should first start the job cluster service:
 
 In order to deploy the job cluster entrypoint run:
 
-`FLINK_IMAGE_NAME=<IMAGE_NAME> FLINK_JOB=<JOB_NAME> FLINK_JOB_PARALLELISM=<PARALLELISM> envsubst < job-cluster-job.yaml.template | kubectl create -f -`
```

Comment:
Why did we remove `FLINK_JOB` here?

Reply:
I found that FLINK_JOB has been removed in commit [753e0c6](https://github.com/dianfu/flink/commit/753e0c618a30a19954761cd75fadba4a89836ae1) and just correct the documentation here. What do you think?

Reply:
Good catch. Now this makes sense.